src/main/java/mate/academy/spring/boot/repository/BookRepository.java

```diff
@@ -1,8 +1,11 @@
 package mate.academy.spring.boot.repository;
 
 import mate.academy.spring.boot.model.Book;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 
 public interface BookRepository extends JpaRepository<Book, Long>, JpaSpecificationExecutor<Book> {
+    Page<Book> findAll(Pageable pageable);
```

Comment:
do you really need this method? Doesn't JpaRepository provide you with this method by default?